graphql/src/main/java/io/jmix/graphql/datafetcher/EntityMutationDataFetcher.java

```diff
@@ -226,4 +282,54 @@ protected void assureCompositionInverseLink(Object parent, MetaClass parentMetaC
             }
         }
     }
+
```

Comment:
Methods below of getting custom editors should be extracted to separate class, together with logic of getting editor bean for exact entity class.